conf/cassandra.yaml

```diff
@@ -2077,6 +2077,42 @@ drop_compact_storage_enabled: false
 # column_value_size_warn_threshold:
 # column_value_size_fail_threshold:
 #
+# Guardrail to warn or fail when writing ascii column values larger than threshold.
+# This guardrail is only applied to the values of regular columns because both the serialized partitions keys and the
+# values of the components of the clustering key already have a fixed, relatively small size limit of 65535 bytes, which
+# is probably lesser than the thresholds defined here.
+# The two thresholds default to null to disable.
```

Comment:
Can you add the clarification when both `column_value_size_warn_threshold` and `column_ascii_value_size_warn_threshold` are enabled? 
For example, `column_value_size_warn_threshold=100B` and `column_ascii_value_size_warn_threshold=200B`. If the incoming write has the `ascii` field size of 123B, should it warn or not?